fe/pom.xml

```diff
@@ -435,6 +435,12 @@ under the License.
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
+                <groupId>org.glassfish</groupId>
+                <artifactId>javax.el</artifactId>
+                <scope>provided</scope>
```

Comment:
Since there are many transitive dependencies, I set it to scoped to provied to avoid packaging this jar